spiceaidocs/docs/components/data-accelerators/sqlite.md

```diff
@@ -21,6 +21,7 @@ datasets:
 The connection to SQLite can be configured by providing the following `params`:
 
 - `sqlite_file`: The filename for the file to back the SQLite database. Only applies if `mode` is `file`.
+- `busy_timeout`: Optional. The SQLite [busy_timeout](https://www.sqlite.org/c3ref/busy_timeout.html) in millisecond for SQLite database file connection. Default is 5000 (milliseconds).
```

Comment:
All other parameters support specifying units, I think we should be consistent and allow to specify this as '5s', '5000ms', etc